.travis.yml

```diff
@@ -0,0 +1,15 @@
+os: osx
+language: objective-c
+osx_image: xcode8.3
+node_js: 5.1.1
+
+before_install:
+  - cd server
+
+install:
+  - npm install
+
+script:
```

Comment:
Ah. Thanks.

I think that running the build step would still be useful, though; not as a prerequisite of running the tests, but as a sanity check, just to have the project be _built_ on a remote machine, so that undocumented local dependencies can't creep into the build process.